### ISSUE TITLE
Dependencies don't get imported automaticly 

### ISSUE THREAD

**Issue by mwillebrands** [chrislove/Hangfire.MySql#6]
Recieving errors about FluentAssertions, Mysql.Data, Linq2DB when using the plugin

**Comment by chrislove** [comment]
I believe this is fixed now - but it's hard to be sure. I've removed and added those packages to the project so many times I'm seeing spots. 

https://github.com/chrislove/Hangfire.MySql/releases/tag/0.0.4
https://www.nuget.org/packages/Hangfire.MySql/

**Comment by vanerleo** [comment]
I don't think it is solved, When installing a nuget for the first time , there is an error about Linq2db
The only way to solve this is to upgrade the all related packages to the latest.

btw i'm running .net 4.6

**Comment by chrislove** [comment]
@vanerleo I've downgraded the .NET version for the project to .NET 4.5 to match Hangfire. If you're still getting errors can you let me know when you get the error (ie. compile-time or run-time) and what the error message is?

**Comment by chrislove** [comment]
Corrected a mistake on the FluentAssertions dependency 3dea1ba1d8431bd343b511d25ec57031ae914752
Seems to work correctly when added to an empty project

**Comment by ssippe** [comment]
I solved this by running...
`PM> Update-Package -reinstall -Project Hangfire.MySql`
